### PR TITLE
Add SoCalGas

### PR DESCRIPTION
**Pull request by johntheholman**
Added json file for the Utility company SoCalGas
Added documentation and 2fa types
Added PNG logo image

### CODE REVIEW

File: entries/s/socalgas.com.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "Southern California Gas Company": {
+    "domain": "socalgas.com",
+    "img": "site.com.png",
+    "tfa": ["sms", "totp"],
+    "documentation": "https://www.socalgas.com/help-center/2-factor-authentication",
+    "regions": ["us"],
+    "categories": ["utilities"]
```
Comment (Carlgo11) [inline entries/s/socalgas.com.json]: Thanks for the pull request @johntheholman.
Could you please use the code style that the rest of the JSON files use? Where `{` and `[` are followed by new lines and `]` `}` are prefixed by new lines.

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
SVG images are preferred over PNGs.
Here's a version using SVG:

File: entries/s/socalgas.com.json (hunk 1 above)
Comment (johntheholman) [reply entries/s/socalgas.com.json]: I will update this. Not sure why the config is not working. Looking into that.

**Review by Carlgo11 — APPROVED** [review]
Thank you! :smiley: 
Your contribution will be visible in around 10 minutes. Sometimes it takes longer due to server and browser cache.

**Comment by johntheholman** [comment]
Thanks @Carlgo11 for such a quick turnaround.

**Pull request merged**